### ISSUE TITLE
Add Cloud.gov sandbox banner to all dev/test/staging Swagger UIs

### ISSUE THREAD

**Issue by david-cvp** [US-EPA-CAMD/PI-1#508]
### Definition of Ready
- [x] Does this meet the INVEST criteria?
- [x] Does this have a clearly defined user?
- [x] Does this have an estimate?
- [x] Does this have acceptance criteria?
- [x] Does this have sufficient context (wireframes, business rules, etc.)?
- [x] Is this traceable to a feature, epic, or PI?

### Acceptance Criteria
As requested by @maheese 
GOAL: to indicate in Swagger UI that the data being presented is a dev system and not PROD data . Need not be banner specifically. 

> Since the UI we are publishing out to Cloud.gov for development purposes is public, we need to alert people that what they are viewing is not a production site and is only used for testing purposes.  This is an example of the banner from the Cloud.gov Rules of Behavior.  It doesn't have to be exactly this but should convey the message.

`<div id="topbanner"><p>EPA Sandbox Environment: The content on this page is not production data and this site is being used for <strong>testing</strong> purposes only.</p></div>`

> Include the following CSS in your header tag:

`<style type="text/css">
#topbanner{position: fixed; z-index: 99999; width:100%; margin:0 auto;padding:0px 5px;text-align:center;background:linear-gradient(to bottom,#520001 0%,#6c0810 100%)}
#topbannerinner{margin:0 auto;width:100%}
#topbanner h3,#topbanner p{color:#FC0}
#topbanner a:link,#topbanner a:visited,#topbanner a:hover{color:#fff}
</style>`

### Definition of Done
- [ ] Have all Acceptance Criteria been agreed to and validated by the scrum team?
- [ ] Has the code been unit tested and peer reviewed?
- [ ] Have the functional tests been executed?
- [ ] Have all defects been reviewed and dispositioned, resolved, or deferred?
- [ ] Has the user story been accepted by the Product Owner?
- [ ] Is the user story ready to be deployed to the staging/integration environment?

**Comment by david-cvp** [comment]
We can split this up for each API once it is ready to develop

Blocked by #722
